ledger/src/blockstore.rs

```diff
@@ -967,6 +967,18 @@ impl Blockstore {
         Ok((newly_completed_data_sets, inserted_indices))
     }
 
+    /**
+     * Range-delete all entries which prefix matches the specified slot and
+     * clear all the related metadata except its next_slots.
```

Comment:
I would specify metadata in the `SlotMeta` struct, since the other column being deleted families are also technically metadata